framework/test/bench/ServiceRegistryTest.cpp

```diff
@@ -184,3 +200,51 @@ BENCHMARK_REGISTER_F(ServiceRegistryFixture, UnregisterServices)
   ->RangeMultiplier(4)
   ->Ranges({ { 1, 1000 }, { 1, 1000 } })
   ->UseManualTime();
+
+BENCHMARK_DEFINE_F(ServiceRegistryFixture, ModifyServices)
+(benchmark::State& state)
+{
+    using namespace std::chrono;
+
+    auto fc = framework->GetBundleContext();
+    auto regCount = state.range(0);
+    auto interfaceCount = state.range(1);
+    auto interfaceMap = MakeInterfaceMapWithNInterfaces(interfaceCount);
+
+    std::vector<ServiceRegistrationBase> regs;
+    for (auto i = regCount; i > 0; --i) {
+        InterfaceMapPtr iMapCopy(std::make_shared<InterfaceMap>(*interfaceMap));
+        auto reg =
+            fc.RegisterService(iMapCopy); // benchmark the call to RegisterService

```

Comment:
the comment says to benchmark this call however it won't be benchmarked since its not in the `for auto _ : state)` loop. is this call really meant to be benchmarked?

Reply:
Nope. Its not meant to be benchmarked.
Removed the comment.